src/main/java/trello/entities/Card.java

```diff
@@ -38,18 +38,18 @@ public void setId(final String id) {
     }
 
     /**
-     * This method is used get name.
-     * @return a string with the name of card.
+     * This method is used get title.
+     * @return a string with the title of card.
```

Comment:
Please, add white space above this line.

Reply:
Solved